dgraph/cmd/zero/run.go

```diff
@@ -90,6 +90,9 @@ instances to achieve high-availability.
 	// --tls SuperFlag
 	x.RegisterServerTLSFlags(flag)
 
+	// Cache flags.
+	flag.Int64("cache_mb", 1024, "Total size of cache (in MB) to be used in Dgraph.")
```

Comment:
Zero doesn't even use this flag. Is this still needed?

Reply:
Will remove this.